src/pages/Pathway/Pathway.js

```diff
@@ -7,8 +7,17 @@ import './Pathway.css';
 import resources from '../../data/resources';
 
 const Pathway = () => {
-    const example_resources = resources[3].links;
-
+    // will probably need props to know when to use the function to create a new pathway, not positive though
+    const [pathway, setPathway] = useState(
+        JSON.parse(localStorage.getItem('pathwayData')) ? JSON.parse(localStorage.getItem('pathwayData')) : resources[2].links
```

Comment:
do we only need the resource index? I wasn't super sure how we would implement the generation of the pathway, but thought it may be taking from several different resource indices...I was probably mistaken.